index.html

```diff
@@ -520,32 +520,22 @@ <h3>Full Name</h3>
             <div class="custom-card-wrapper col-12 col-md-6 col-lg-4">
               <div class="custom-card">
                 <div class="avatar-wrapper">
-                  <img src="img/placeholder.jpeg" alt="" class="avatar" />
+                  <img src="img/CristianBahnaru.jpeg" alt="Poza Cristian Bahnaru" class="avatar" />
                 </div>
                 <div>
-                  <h3>Full Name</h3>
+                  <h3>Cristian Aurelis Bahnaru</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    I work well with others and value their presence and contributions when working on collaborative projects. After all, we all need to be able to work together to succeed.   I am also very approachable and always willing to help and support my coworkers when needed..I have an passion for Web Development and programming languages, learn new technologies and improve my knowledge.
                   </p>
-                  <p>
-                    Quidem, corrupti perspiciatis aliquid dicta cum odio
-                    sapiente tempora quas incidunt adipisci alias praesentium et
-                    excepturi error maxime aspernatur sint qui doloremque
-                    distinctio. Ex, unde.
+                   <p>My favorite motto is :</p>
+                      <p>"Inspire people,Empower people,Share Vision,Lead Change,Make it Happen"</p>
                   </p>
                 </div>
                 <div class="contact">
-                  <a href="#"
-                    ><img src="assets/linkedin.svg" alt="linkedin"
-                  /></a>
-                  <a href="#"><img src="assets/github.svg" alt="github" /></a>
-                  <a href="#"><img src="assets/codepen.svg" alt="codepen" /></a>
-                  <a href="#"
-                    ><img src="assets/codesandbox.svg" alt="codesandbox"
-                  /></a>
+                  <a href="https://www.linkedin.com/in/cristian-aurelis-bahnaru-098822144/"><img src="assets/linkedin.svg" alt="linkedin"/></a>
```

Comment:
Have you tried generating a custom link for LinkedIn? It would be easier to include it in your CV (e.g. `https://www.linkedin.com/in/cristian-aurelis-bahnaru` instead of `https://www.linkedin.com/in/cristian-aurelis-bahnaru-098822144`)